### ISSUE TITLE
Understat data confusion

### ISSUE THREAD

**Issue by ThomasAllan** [vaastav/Fantasy-Premier-League#48]
Hello, 

Bit confused with the understat data. There seem to be duplicate files, see;
1. Fantasy-Premier-League/data/2019-20/understat_Arsenal.csv
2. Fantasy-Premier-League/data/2019-20/understat/understat_Arsenal.csv

They both seem to have the same headers but the files in the understat folder have header names in alphabetical order where those in the root season folder are in a random order. 

Also, using the Arsenal file as an example, the file in the understat folder has less rows than that in the root season folder indicating missing game weeks. 

To summarise the understat files in the 2019-20 folder look correct (apart from alphabetical headers) but I would expect them to be in the understat folder not in the root?

**Comment by vaastav** [comment]
You are correct. The data was just duplicated. As far as I can tell there was no difference between the 2 files. The issue is fixed now and the correct data is in the understat folder and not in the root folder.
